helm/values-prod.yaml

```
@@ -0,0 +1,19 @@
+image:
+  repository: selcpacr.azurecr.io/selfcareexternalapi
```

here should go the repo name without "-" (it's configured in gitops project)
```suggestion
  repository: selcpacr.azurecr.io/selfcareexternalapibackend
```